### ISSUE TITLE
Question: No module named 'aws_lambda_logging'

### ISSUE THREAD

**Issue by opiuman** [awslabs/aws-lambda-powertools#2]
Tried the logging code from README and got the error:

`{
  "errorMessage": "Unable to import module 'service': No module named 'aws_lambda_logging'",
  "errorType": "Runtime.ImportModuleError"
}`

The lambda runtime is Python 3.7.

**Comment by heitorlessa** [comment]
Hmmm odd. tests weren’t failing when I made that change - Here’s where the
issue might be

https://github.com/awslabs/aws-lambda-powertools/blob/develop/python/aws_lambda_powertools/logging/logger.py

We might need to do an explicit “import .aws_lambda_logging”, or PyPi
doesn’t have the latest.

I’ll looking into that as soon as I’m back from holidays (Jan 8th).

Thanks for reporting it

**Comment by nmoutschen** [comment]
I just had the same issue on my side, with the following error message in my Lambda logs:

```
[ERROR] Runtime.ImportModuleError: Unable to import module 'main': No module named 'aws_lambda_logging'
```

This was solved by explicitely adding `aws-lambda-logging` in my requirements.txt.

**Comment by heitorlessa** [comment]
I’ve just come back from holidays and will publish a fix by end of this
week.

No need to add that library in the requirements, it’s a bad relative import
in the logger.py

**Comment by heitorlessa** [comment]
Fix has been published -- 0.3.1 is now fully working. Thanks for reporting @opiuman !
